examples/simulate_browser.rs

```diff
@@ -145,7 +145,7 @@ fn add_service(client        : std::sync::Arc<std::sync::Mutex<safe_client::clie
                                              None));
 
     let file_helper = safe_nfs::helper::file_helper::FileHelper::new(client.clone());
-    let mut writer = try!(file_helper.create(HOME_PAGE_FILE_NAME.to_string(), vec![], dir_listing));
+    let mut writer = try!(file_helper.create(HOME_PAGE_FILE_NAME.to_string(), vec![], dir_listing.0));
```

Comment:
The variable name dir_listing and operations such as `dir_listing.0` does not make sense. Use de-structuring to make it sensible:
`let (created_dir_listing, _) = try!(dir_helper.create(......` etc